### DISCUSSION TITLE
integrations/ibm-watsonx/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#17967] [Docs]
# integrations/ibm-watsonx/

Dive into our detailed integration guide on using IBM Watson to train a YOLO11 model. Uncover key features and step-by-step instructions on model training.

https://docs.ultralytics.com/integrations/ibm-watsonx/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for your interest in Ultralytics 🚀! It's great to see your enthusiasm for exploring integrations like IBM Watsonx. For new users, we highly recommend checking out the [Docs](https://docs.ultralytics.com) for comprehensive guides, including additional integration examples and tutorials.

If your post is related to a ❓ question or assistance needed regarding this integration, please provide more details about what you’re looking for. If you’re encountering an issue or a 🐛 bug, kindly share a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us investigate further.

Additionally, if you're training custom models, we suggest following our [Model Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) to achieve the best results, and feel free to include any relevant details such as logs, dataset parameters, or challenges you're facing. 🎯

To try out YOLO using various environments with IBM Watsonx or otherwise, take advantage of these verified setups for a smooth experience:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

You can also keep your system up-to-date by upgrading to the latest `ultralytics` package in a [**Python>=3.8**](https://www.python.org/) environment. Simply run:

```bash
pip install -U ultralytics
```

💡 For real-time discussions or additional help, join our [Discord](https://discord.com/invite/ultralytics) community 🎧. If you prefer more traditional discussions, visit [Discourse](https://community.ultralytics.com) or our [Subreddit](https://reddit.com/r/Ultralytics).

Finally, if you haven't already, explore the [IBM Watsonx integration guide](https://docs.ultralytics.com/integrations/ibm-watsonx/) for step-by-step instructions and key features.

An Ultralytics engineer will follow up soon to assist further! 😊
